programs/client/Client.cpp

```diff
@@ -779,6 +779,11 @@ bool Client::processWithFuzzing(const String & full_query)
             if (fuzz_step > 0)
             {
                 fuzzer.fuzzMain(ast_to_process);
```

Comment:
Also, `fuzzMain` can be extended to accept two optional parameters - stdout and stderr buffers. This way debugging output can be preserved.